appyters/DrugBank/README.md

```diff
@@ -0,0 +1,6 @@
+# DrugBank
+
+Date Created: 6-20
```

Comment:
This isn't necessary and will appear redundant because we add it automatically.

Reply:
So I meant to remove the date in the README as opposed to the appyter where you can have it or not

Reply:
Got it. I'll look into that in the future.